packages/react-components/react-avatar/src/components/AvatarGroupOverflow/AvatarGroupOverflow.test.tsx

```diff
@@ -1,18 +1,63 @@
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import { AvatarGroupItem } from '../AvatarGroupItem/AvatarGroupItem';
 import { AvatarGroupOverflow } from './AvatarGroupOverflow';
+import { avatarGroupOverflowClassNames } from './useAvatarGroupOverflowStyles';
 import { isConformant } from '../../common/isConformant';
+import { RenderResult } from '@testing-library/react';
+
+// testing-library's queryByRole function doesn't look inside portals
+function queryByRoleDialog(result: RenderResult) {
+  const dialogs = result.baseElement.querySelectorAll('*[role="dialog"]');
+  if (!dialogs?.length) {
+    return null;
+  } else {
+    expect(dialogs.length).toBe(1);
+    return dialogs.item(0) as HTMLElement;
+  }
+}
+
+const getOverflowSurfaceElement = (result: RenderResult) => {
```

Comment:
IMO testing overflow in unit test runners like just with JSDOM is not very useful. You might want to consider using cypress to test this on the browser directly

Reply:
I move the tests to cypress in a follow up PR, but here's an [issue](https://github.com/microsoft/fluentui/issues/24173) to track it.